### ISSUE TITLE
Feature Request - Apply for bulk attendance request 

### ISSUE THREAD

**Issue by BhuviTheDataGuy** [horilla-opensource/horilla#180]
Sometimes people wont do daily check-in checkout(some top executives) , they do create attendance request on the last day on the month and apply it for date ranges.

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
The feature for bulk attendance request has been updated in the codebase.
Please check and let us know if you have any more concerns related to it.

With Regards,
**Team Horilla**

**Comment by BhuviTheDataGuy** [comment]
I tried to create a new request, but its not working.

![image](https://github.com/horilla-opensource/horilla/assets/47310439/d7f6d50b-5500-4554-87d8-6cdae154a5cb)

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
Were you able to test out the solution ?

With Regards,
**Team Horilla**

**Comment by horilla-opensource** [comment]
> I tried to create a new request, but its not working.
> 
> ![image](https://private-user-images.githubusercontent.com/47310439/334523539-d7f6d50b-5500-4554-87d8-6cdae154a5cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzOTk0MzYsIm5iZiI6MTcxOTM5OTEzNiwicGF0aCI6Ii80NzMxMDQzOS8zMzQ1MjM1MzktZDdmNmQ1MGItNTUwMC00NTU0LTg3ZDgtNmNkYWUxNTRhNWNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI2VDEwNTIxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwMmRhN2Q2MGE2Mzg4MTFkNWYzNDQ5ZmI0ZTEwZjBiMTAwNzhmZTc1N2YwYWFkOTAxZjI3OTI2ZjJmYzdkNWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5c2fgqV50UXMh1XqTwKnEH6AhFv7soW2n3kjY82dxaw)

Is this the error you are still facing?

With Regards,
**Team Horilla**

**Comment by BhuviTheDataGuy** [comment]
Yes, same error.

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
Can you please confirm if there is any holidays coming in between the requested days ?

With Regards,
**Team Horilla**

**Comment by BhuviTheDataGuy** [comment]
Yes, I created the request from day 1 to day 30

**Comment by BhuviTheDataGuy** [comment]
The idea is, we can request any date range, if any approved leave, weekend and company leaves will be automatically taken.
And in between if I already checked in it will be replaced by the new request basis.

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
Sorry, I have missed the reply.
I get the point and update you with the solution.

With Regards,
**Team Horilla**

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
We have checked the system with existing database and fresh database. Its working fine in our case.
Can we jump on a meet to discuss the same within your database?
Please let me know your availability .

With Regards,
**Team Horilla**

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
Can we close this issue now?

With Regards,
**Team Horilla**

**Comment by BhuviTheDataGuy** [comment]
Help required.

Not able to apply for bulk attendance request. I tried for 1 day as well, but its not working.

![image](https://github.com/user-attachments/assets/26f7abf0-7ddf-4734-9a09-b414fa137782)

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
It was working fine before right?

With Regards,
**Team Horilla**

**Comment by BhuviTheDataGuy** [comment]
No, it was not, may be for other users. But I'm the super user, its not working

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
We have tested it out with all types of employees in a fresh db and also an existing live database. 
Could you please check if there is any extra data provided in the console which could help us pin pin  point the issue ?

With Regards,
**Team Horilla**

**Comment by horilla-opensource** [comment]
Hi @BhuviTheDataGuy ,
We are closing this issue due to inactivity. It seems that there hasn't been any recent activity or discussion, and we believe it may have been resolved or is no longer relevant. 
If you feel this issue is still important and should be addressed, please feel free to reopen it or create a new issue with updated details. 

With Regards,
**Team Horilla**
